src/test/java/com/sleekbyte/tailor/utils/CharFormatUtilTest.java

```diff
@@ -88,4 +88,16 @@ public void testStartsWithAcronym() {
         assertFalse(CharFormatUtil.startsWithAcronym("xURLS"));
         assertFalse(CharFormatUtil.startsWithAcronym("shieldPROGRAMMEmarvel"));
     }
+
+    @Test
+    public void testBacktickEscapedIdentifier() {
+        // Backtick(s) are not part of the identifier
+        assertTrue(CharFormatUtil.unescapeIdentifier("``").isEmpty());
+        assertTrue(CharFormatUtil.unescapeIdentifier("").isEmpty());
+        assertTrue(CharFormatUtil.unescapeIdentifier("`s`").equals("s"));
+        assertTrue(CharFormatUtil.unescapeIdentifier("`self`").equals("self"));
+        assertFalse(CharFormatUtil.unescapeIdentifier("`self").equals("self"));
```

Comment:
`assertFalse` is kind of weird here (it's like saying we expect that it should be anything but `self`). Can we change this to `assertEqual` (and the following `assertFalse`(s) as well)?